src/packages/base_components/AppBars.tsx

```diff
@@ -0,0 +1,62 @@
+import {Appbar as AppBar, useTheme} from 'react-native-paper';
+import React from 'react';
+
+export function MainDrawerAppBar({onActionMenuPress, title}: {onActionMenuPress: () => void; title: string}) {
+  return (
+    <AppHeader>
+      <AppBar.Action
+        tvParallaxProperties={undefined}
+        hasTVPreferredFocus={false}
+        onPress={onActionMenuPress}
+        icon={'menu'}
+      />
+      <AppBar.Content title={title} />
+    </AppHeader>
+  );
+}
+
+export function MainStackAppBar({
+  headerLeft,
+  headerRight,
+  title,
+}: {
+  title: string;
+  headerLeft?: React.ReactNode;
+  headerRight?: React.ReactNode;
+}) {
+  return (
+    <AppHeader>
+      {headerLeft}
+      <AppBar.Content title={title} />
+      {headerRight}
+    </AppHeader>
+  );
+}
+
+export function AppHeader({children}: {children: React.ReactNode}) {
+  const theme = useTheme();
+
+  return <AppBar.Header style={{backgroundColor: theme.colors.background}}>{children}</AppBar.Header>;
+}
+
+export function DashboardAppBar({
+  onActionLeftPress,
+  onContentPress,
+  subtitle,
+}: {
+  onActionLeftPress: () => void;
+  onContentPress: () => void;
+  subtitle?: React.ReactNode;
+}) {
+  return (
+    <AppHeader>
+      <AppBar.Action
+        tvParallaxProperties={undefined}
```

Comment:
We can probably just drop the prop since you are passing `undefined`

Reply:
as mentioned it previous comment, would show an ts issue without it. couldn't find a better way to fix this.